src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/builder/AbstractTestBuilder.groovy

```diff
@@ -43,6 +43,14 @@ abstract class AbstractTestBuilder implements Serializable {
         this.context = context
     }
 
+    ExecutionConfig getExecutionConfig() {
+        return executionConfig? new ExecutionConfig(executionConfig) : null
```

Comment:
format -> `return executionConfig ? new ExecutionConfig(executionConfig) : null`
and L51